pkg/svcat/service-catalog/binding_test.go

```diff
@@ -312,4 +312,106 @@ var _ = Describe("Binding", func() {
 			Expect(deletedBindings[sb2.Name]).To(Equal(sb2.Namespace))
 		})
 	})
+
+	Describe("RemoveFinalizersForBinding", func() {
+		It("Calls the generated v1beta1 put method with the passed in binding", func() {
+			err := sdk.RemoveFinalizersForBinding(*sb)
+			Expect(err).NotTo(HaveOccurred())
+
+			actions := svcCatClient.Actions()
+			Expect(actions[0].Matches("update", "servicebindings")).To(BeTrue())
+			Expect(actions[0].(testing.UpdateActionImpl).Object.(*v1beta1.ServiceBinding).ObjectMeta.Name).To(Equal(sb.ObjectMeta.Name))
+			Expect(actions[0].(testing.UpdateActionImpl).Object.(*v1beta1.ServiceBinding).ObjectMeta.Namespace).To(Equal(sb.ObjectMeta.Namespace))
```

Comment:
These tests are great! Only other thing I could ask for us somehow asserting that the update action is removing the finalizers, but I don't actually know how to do that.